index.html

```diff
@@ -2120,588 +2138,378 @@ <h3>Thing Root Object</h3>
+        "status": {
+            "titles": {
+                "en": "Status",
+                "de": "Zustand",
+                "ja": "状態",
+                "zh-Hans": "状态",
+                "zh-Hant": "狀態" },
+            "type": "object",
+            ...
             "forms": [...]
         }
     },
     ...
+}
 </pre>
-</aside>
- 
-
-</section> <!-- end of id="action-serialization-json"-->
-
-<section id="event-serialization-json">
-<h3><code>events</code></h3>
-
-  <p>
-    In a <code>Thing</code> instance, the value assigned to <code>events</code>
-    is a map of instances of <code>EventAffordance</code>.
-    <span class="rfc2119-assertion" id="td-events">
-    All name-value pairs of a <a>Map</a> of <code>EventAffordance</code> instances
-    MUST be serialized as members of the JSON object that results from serializing the <a>Map</a>;
-    the name of a pair MUST be serialized as a JSON string and the value of the pair, an
-    instance of <code>EventAffordance</code>, MUST be serialized as a JSON object.
-  </span></p>
-
-  <p><span class="rfc2119-assertion" id="td-event-names">
-    All name-value pairs of an instance of <code>EventAffordance</code>,
-    where the name is a <a>Vocabulary Term</a> included in (one of) the <a>Signatures</a> of
-    <code>EventAffordance</code> or <code>InteractionAffordance</code>,
-    MUST be serialized as members of the JSON object that results from
-    serializing the <code>EventAffordance</code> instance, with the
-    <a>Vocabulary Term</a> as name.
-  </span></p>
-
+    
   <p>
-  <span class="rfc2119-assertion" id="td-event-objects">
-    The values assigned to <code>subscription</code>,
-    <code>data</code>, and <code>cancellation</code>
-    in an instance of <code>EventAffordance</code>
-    MUST be serialized as JSON objects.
-  </span>
-  They rely on the the <a>Class</a> <a href="#dataschema" class="sec-ref"><code>DataSchema</code></a>,
-  whose serialization is defined in <a href="#data-schema-serialization-json"></a>.
-  </p>
-
-  <p><span class="rfc2119-assertion" id="td-event-arrays">
-  The value assigned to <code>forms</code> in an instance of <code>EventAffordance</code>
-  MUST be serialized as a JSON array
-  containing one or more JSON object serializations as defined in <a href="#form-serialization-json"></a>.
-  </span></p>
-
-  <p>
-  A TD snippet of an Event object is given below:
+  TD instances may also combine the use of <code>title</code> and <code>description</code>
+  with <code>titles</code> and <code>descriptions</code>.
+  <span class="rfc2119-assertion" id="td-title-description">
+  When <code>title</code> and <code>titles</code> or
+  <code>description</code> and <code>descriptions</code>
+  are present within the same JSON object,
+  the values of <code>title</code> and <code>description</code> MAY be seen as the default text.
+  </span>
+  These values underly the default language, which is usually set by the creator of the Thing Description instance.
```

Comment:
"underly" is not a word.   If you meant "underlay", that does not mean the right thing here.   I think you want to say "These values are expressed using the default language"

Reply:
The the language of the default text is indicated by the default language,